dist/preview release/what's new.md

```diff
@@ -89,6 +89,8 @@
 - Add support to update inspector when switching to a new scene ([belfortk](https://github.com/belfortk))
 - Hex Component for Hex inputs on layer masks. ([msDestiny14](https://github.com/msDestiny14))
 - View & edit textures in pop out inspector using tools such as brush and floodfill. Supports region selection, individual channel editing, mipmap previews, and resizing. ([DarraghBurkeMS](https://github.com/DarraghBurkeMS))
+- Added mesh debug tools to display bone influence weights and bone indices maps. ([Pryme8](https://github.com/Pryme8))
+- Added more functionality an options to the skeleton debug panel. ([Pryme8](https://github.com/Pryme8))
```

Comment:
nit: I think it should be "and" instead of "an"?

Reply:
Will fix right now